src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1729,6 +1729,7 @@ internal enum ErrorCode
         ERR_DefaultLiteralNoTargetType = 8716,
 
         ERR_ReAbstractionInNoPIAType = 8750,
+        ERR_InternalError = 8751,
```

Comment:
This should probably go past the C# 8 region. Additionally, it feels like we should add an assert in the Diagnostic creation code that we never see this code, for the inevitable future case that adds one of these without an accompanying assert.